var/spack/repos/builtin/packages/easybuild/package.py

```diff
@@ -12,10 +12,16 @@ class Easybuild(PythonPackage):
     """
 
     homepage = 'http://hpcugent.github.io/easybuild/'
-    url      = 'https://pypi.io/packages/source/e/easybuild/easybuild-3.1.2.tar.gz'
+    url      = 'https://pypi.io/packages/source/e/easybuild/easybuild-4.0.0.tar.gz'
+    maintainers = ['boegel']
 
+    version('4.0.0', sha256='21bcc1048525ad6219667cc97a7421b5388068c670cabba356712e474896de40')
     version('3.1.2', 'c2d901c2a71f51b24890fa69c3a46383')
```

Comment:
What about checksums for older `3.x` versions?

Reply:
I can add them if you want. I only added the latest version so I could get Python 3 support 😄

Reply:
Well, up to you, not sure if it's worth the effort, since there's no point in preferring an EasyBuild v3.x over 4.0.0 or newer.